frontend/src/components/Footer.jsx

```
@@ -0,0 +1,78 @@
+              </a>
+              <a href="#" className="text-gray-200 hover:text-white transition-colors">
+                <Mail size={20} />
+              </a>
+            </div>
+          </div>
+          
+          {/* Quick Links */}
+          <div>
+            <h4 className="font-semibold text-lg mb-4">Quick Links</h4>
+            <ul className="space-y-2">
+              <li><a href="#" className="text-gray-200 hover:text-white transition-colors">Home</a></li>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)